src/collectors/windows.plugin/integrations/ms_sql_server.md

```diff
@@ -88,7 +88,11 @@ Metrics:
 | mssql.instance_bufman_iops | read, written | pages/s |
 | mssql.instance_bufman_checkpoint_pages | log | pages/s |
 | mssql.instance_bufman_page_life_expectancy | life_expectancy | seconds |
+| mssql.instance_bufman_lazy_write | lazy_write | Lazy writes/sec |
+| mssql.instance_bufman_page_lookups | page_lookups | Page lookups/sec |
 | mssql.instance_memmgr_server_memory | memory | bytes |
+| mssql.database_readonly | readonly | boolean |
+| mssql.database_state | state | boolean |
```

Comment:
Wrong too

Reply:
@thiagoftsm please fix

Reply:
Fixed in https://github.com/netdata/netdata/pull/20815